ReactWindows/ReactNative/Modules/Vibration/VibrationModule.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using ReactNative.Bridge;
+using Windows.Phone.Devices.Notification;
+
+namespace ReactNative.Modules.Vibration
+{
+    /// <summary>
+    /// Represents the module for vibration.  Note this is only available on Windows Mobile.
+    /// </summary>
+    public class VibrationModule : NativeModuleBase
+    {
+        private readonly bool _isMobile;
+
+        internal VibrationModule() : this(IsMobile())
```

Comment:
internal APIs still should be documented (and not sure why we need this to be internal).  If there are other modules with internal constructors, we should do a pass to clean that up.